mondrian-generator.js

```diff
@@ -113,23 +161,46 @@ function fillContextSquares(context, x, y) {
   return context
 }
 
-function makeMondrianImg(shouldSave) {
-  const { x, y } = getLinePositions()
+const makeArtistImgFuncs = {
+  Mondrian: (artistName, shouldSave) => {
```

Comment:
This kind of object mapper, when combined with `makeArtistImg()`, would be a really teachable moment for young coders. It would educate them in abstracting basic operations into higher-order controller functions.